docs/settingUp.md

```diff
@@ -1,333 +1,86 @@
-   and update the `test.chromedriver.path` value in `test.properties`.
-   The site will also inform the versions of Chrome that can be used with the driver.
-
-2. Before running the test suite, both the server and the test environment 
-   should be using the UTC time zone.
-   
-   Here is the procedure:
-    
-    a. Stop the dev server, if it is running already.
-
-    b. Specify timezone as a VM argument: 
-       * Go to the `run configuration` Eclipse created when you started the dev server
-        (`Run → Run configurations ...` and select the appropriate one).
-       * Click on the `Arguments` tab and add `-Duser.timezone=UTC` to the `VM arguments` text box.
-       * Save the configuration for future use: Go to the `Common` tab (the last one) 
-       and make sure you have selected `Save as → Local file` and 
-       `Display in favorites menu →  Run, Debug`.
-
-    c. Start the server again using the _run configuration_ you created in
-       the previous step..<br>
-   
-3. Run tests. <br>
-   Test can be run using the configurations available under the green `Run` button
-   in the Eclipse toolbar. There are several configurations that are provided by default.
-   These are:
-
-      * `All tests` - This runs the files `src/test/testng-travis.xml` as well as `src/test/testng-local.xml`.
-      * `Travis tests` - This runs the file `src/test/testng-travis.xml`. It contains all the tests that are run by Travis.
-      * `Local tests` - This runs the file `src/test/testng-local.xml`. It contains all the tests that need
-                         to be run locally by developers. `Dev green` mean passing all the tests in this configuration.
-      * `Staging tests` - This runs a subset of the tests in `src/test/testng-travis.xml`.
-                           This is run before deploying to a staging server.
-
-   New developers should at least run the `Local tests` and have all of them passing on their local environments.
-   Running `Travis tests` and getting everything passing is also recommended.
+1. Add a remote name (e.g `upstream`) for the original repo for your repo to keep in sync with.
+   ```sh
+   git remote add upstream https://github.com/TEAMMATES/teammates.git
+   ```
 
-   Additionally, configurations that run the tests with `GodMode` turned on are also provided.
-   More info on this can be found [here](godmode.md). Sometimes, Eclipse does not show
-   these options immediately after you set up the project. 'Refreshing' the project should fix that. 
+More information can be found at [this documentation](https://help.github.com/articles/fork-a-repo/).
 
-   When running the test cases, if a few cases fail (this can happen due to timing issues),
-   run the failed cases using the `Run Failed Test` icon in the TestNG tab in Eclipse
-   until they pass. 
+## Step 3: Set up project-specific settings and dependencies
 
+1. Run this command to download the correct version Google App Engine SDK as used in the project:
```

Comment:
Yup this would be helpful. Even it took time for me to figure it out where to run it, on git bash or cmd.